src/go/pt-mongodb-index-check/README.md

```diff
@@ -0,0 +1,48 @@
+# pt-mongodb-index-check
```

Comment:
is this tool here intentionally or it should be in a separate PR ?

Reply:
There is another PR but it is in review. https://github.com/percona/percona-toolkit/pull/537
I wanted to prepare the release branch including this new tool.